src/index.ts

```diff
@@ -860,6 +908,22 @@ class Piscina extends EventEmitterAsyncResource {
   static get Piscina () {
     return Piscina;
   }
+
+  static move (val : any) {
+    if (!isTransferable(val)) {
+      if (types.isAnyArrayBuffer(val)) {
+        val = new DirectlyTransferable(val);
+      } else if ((types as any).isArrayBufferView(val)) {
```

Comment:
Opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44769 to fix this, btw